src/sagemaker_xgboost_container/algorithm_mode/train.py

```diff
@@ -42,6 +47,8 @@ def get_validated_dmatrices(train_path, validate_path, content_type, csv_weights
     :param content_type: Content type of data. Supports 'libsvm' or 'csv'
```

Comment:
I noticed I didn't update this comment to include parquet and recordio-protobuf. Can you quickly fix this?

Reply:
Sure, will do.